src/website/views/components/sidenav/_examples.pug

```diff
@@ -483,6 +483,245 @@ h3 Inverse
 
       <script>chi.sidenav(document.getElementById('sidenav-1'));</script>
 
+h3 Open on hover
+- example +=1
+.example.-mb--3
+  .-d--flex(style='overflow: hidden;')
+    aside.chi-sidenav.-flex--shrink0#example-open-on-hover
```

Comment:
This is rendering with the base theme but the code sample below has an `-inverse` class and renders in the inverse theme. Need to add the -inverse class here.

```suggestion
    aside.chi-sidenav.-inverse.-flex--shrink0#example-open-on-hover
```